core/deadline.go

```diff
@@ -45,10 +45,16 @@ type Deadliner interface {
 	// C returns the same read channel every time and contains deadlined duties.
 	// It should only be called by a single goroutine.
 	C() <-chan Duty
+
+	// AddedChan returns the same read channel every time and contains a bool which is set to
+	// true when call to Add(duty) succeeds. It should only be called by a single goroutine.
+	AddedChan() <-chan bool
```

Comment:
we decided to return bool from Add function. Not adding a new method. remove this

Reply:
done!